### PR TITLE
Ycp weight

### PR DESCRIPTION
**Pull request by yuricarvalhop**
#592 #588 #580 #578 #574 #568 #561

### CODE REVIEW

File: app/routes/home/user/edit.js [hunk 1]
```diff
@@ -12,16 +12,20 @@ export default Route.extend(AuthenticatedRouteMixin,{
...
     controller.set('changePassword', false);
     controller.set('modal', false);
+    controller.set('firstName', this.get('currentModel.firstName'));
+    controller.set('lastName', this.get('currentModel.lastName'));
   },
 
   actions: {
     save(){
       $('span.has-error').removeClass('invisible');
+      this.set('currentModel.firstName', this.controller.firstName);
```
Comment (mgonc) [inline app/routes/home/user/edit.js]: I think it's better use this.controller.get('firstname')

Comment (yuricarvalhop) [reply app/routes/home/user/edit.js]: Ok, gonna wait for his answer, then change this commit

**Comment by yuricarvalhop** [comment]
Changes made.

File: app/components/forms/form-range.js [hunk 2]
```diff
@@ -4,15 +4,18 @@ import FormComponent from 'citron/lib/form-component';
 export default FormComponent.extend({
   classNameBindings: ['error'],
   error: computed('field.error', function(){ return this.get('field.error') }),
+  init(){
```
Comment (mgonc) [inline app/components/forms/form-range.js]: why this is necessary?

File: app/routes/home/user/edit.js [hunk 3]
```diff
@@ -12,16 +12,20 @@ export default Route.extend(AuthenticatedRouteMixin,{
     return this.store.findRecord('user', params.id);
   },
 
-  setupController(controller){
+  setupController(controller, currentModel){
```
Comment (mgonc) [inline app/routes/home/user/edit.js]: Here you can use 
setupController(controller, model){
...
controller.set('firstName', model.get('firstName'));
}

don't need to use this.get('currentModel.firstName'));
￼

File: app/components/forms/form-range.js (hunk 2 above)
Comment (yuricarvalhop) [reply app/components/forms/form-range.js]: Forgot this code. Removed it already

File: app/routes/home/user/edit.js (hunk 3 above)
Comment (yuricarvalhop) [reply app/routes/home/user/edit.js]: Done

**Pull request merged**